src/components/Mailbox.vue

```diff
@@ -15,6 +15,20 @@
 			:slow-hint="t('mail', 'Indexing your messages. This can take a bit longer for larger folders.')" />
 		<EmptyMailboxSection v-else-if="isPriorityInbox && !hasMessages" key="empty" />
 		<EmptyMailbox v-else-if="!hasMessages" key="empty" />
+		<template v-else-if="hasGroupedEnvelopes && !isPriorityInbox">
+			<div v-for="[label, group] in Object.entries(groupEnvelopes)" :key="label">
+				<SectionTitle class="section-title" :name="label" />
+				<EnvelopeList :account="account"
+					:mailbox="mailbox"
+					:search-query="searchQuery"
+					:envelopes="group"
+					:loading-more="false"
+					:load-more-button="false"
+					:skip-transition="skipListTransition"
+					@delete="onDelete"
+					@load-more="loadMore" />
```

Comment:
Don't forget about the load more button ;)

Reply:
we removed them on purpose, if thats what you mean

Reply:
Ah, i see now :)